app/views/hyrax/dashboard/_sidebar.html.erb

```diff
@@ -0,0 +1,155 @@
+<% menu = Hyku::MenuPresenter.new(self) %>
```

Comment:
If we're open to the "Account Settings" being brought out a level instead of being nested under "Settings" then we could avoid the override using https://github.com/samvera/hyrax/pull/3878.  Otherwise we're left at overriding or doing more work within Hyku/Hyrax to allow for more control over changing the sidebar.

Reply:
I think you need to speak to Tom. Let me know the decision reached.

Reply:
If that's the case, then I think this should come in as is and can be changed later.  For now it shouldn't block the PR.